xtas/tasks/_alpino.py

```diff
@@ -78,7 +78,7 @@ def parse_raw(tokens):
     return parse
 
 
-def interpret_parse(parse):
```

Comment:
I don't see any easy simplifications, but this function is pretty difficult to read.